backend/papersfeed/migrations/0009_subscription.py

```diff
@@ -17,18 +17,18 @@ class Migration(migrations.Migration):
             name='Subscription',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('recipient_user_id', models.IntegerField()),
+                ('creation_date', models.DateTimeField(auto_now_add=True, verbose_name='Creation Date')),
+                ('modification_date', models.DateTimeField(auto_now=True, verbose_name='Modification Date')),
                 ('verb', models.CharField(max_length=255)),
                 ('action_object_object_id', models.CharField(blank=True, max_length=255, null=True)),
                 ('target_object_id', models.CharField(blank=True, max_length=255, null=True)),
-                ('timestamp', models.DateTimeField(auto_now=True)),
                 ('action_object_content_type', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='subscribe_action_object', to='contenttypes.ContentType')),
                 ('actor', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='subscribe_actor', to=settings.AUTH_USER_MODEL)),
                 ('target_content_type', models.ForeignKey(blank=True, null=True, on_delete=django.db.models.deletion.CASCADE, related_name='subscribe_target', to='contenttypes.ContentType')),
             ],
             options={
                 'db_table': 'swpp_subscription',
-                'ordering': ('-timestamp',),
+                'ordering': ('creation_date',),
```

Comment:
Why does this part is not `-creation_date`?

Reply:
fixed, maybe the change was not reflected while doing testing.